internal/controllers/reconciler.go

```
@@ -0,0 +1,449 @@
+		if errors.Is(err, context.Canceled) {
+			c.logger.InfoContext(ctx, "Sync finished")
+			return
+		}
+		if err != nil {
+			c.logger.ErrorContext(
+				ctx,
+				"Sync failed",
+				slog.Any("error", err),
+			)
+		}
+		c.sleepAfterSync(ctx)
```

golangci-lint (1.64.8)

Error return value of `c.sleepAfterSync` is not checked

(errcheck)